templates/nutrition_facts_table.tt.html

```diff
@@ -0,0 +1,79 @@
+[% IF comparisons.defined %]
```

Comment:
I think it's very important to add a bit more comments, in particular here where most of the ids are generated. Both HTML and Perl comments. But it can be the job of front-end devs.
```
<!-- Nutrition facts table -->
[% IF comparisons.defined %]
<!-- Comparison to category block -->
```

Reply:
Good idea.

Reply:
```suggestion
<!-- Nutrition facts table -->
[% IF comparisons.defined %]
<!-- Comparison to category block -->
```

Reply:
Right.